.changesets/maint_geal_always_create_traffic_shaping.md

```diff
@@ -0,0 +1,5 @@
+### Always instanciate the traffic shaping plugin ([Issue #3327](https://github.com/apollographql/router/issues/3327))
```

Comment:
Isn't this a fix? I mean, the traffic shaping plugin should always have been enabled, so it feels like we are fixing a problem here.

Reply:
it was supposed to be a fix, because I initially thought it was only instantiated when configured. But after actually testing it, it was always active, so this is mainly a refactor that removes the `Either` around supergraph and subgraph services

Reply:
ok something was wrong with my manual tests, but it was caught by the unit tests. Now with @SimonSapin's recent changes to plugin ordering, I made sure traffic shaping is actually mandatory